step.yml

```diff
@@ -166,9 +201,8 @@ outputs:
   - BITRISE_TEST_BUNDLE_ZIP_PATH:
     opts:
       title: "The built test directory and the built xctestrun file compressed as a single zip"
-  - BITRISE_XCODE_BUILD_RAW_RESULT_TEXT_PATH:
+  - BITRISE_XCODEBUILD_LOG_PATH:
```

Comment:
Do we intend to change this output?

Reply:
I tried to reason about it in the PR description/Decisions section:

**xcodebuild log file path output key renamed**

The step.yml stated that `BITRISE_XCODE_BUILD_RAW_RESULT_TEXT_PATH` will be exported, will the Step exported it under `BITRISE_XCODE_RAW_RESULT_TEXT_PATH` key.

To resolve this issue I introduced `BITRISE_XCODEBUILD_LOG_PATH` output key for the xcodebuild log file path. This way the output became similar to what we have in xcode-test step: https://github.com/bitrise-steplib/steps-xcode-test/blob/master/step.yml#L370-L380

Reply:
agreed to keep the output exported by the step: BITRISE_XCODE_RAW_RESULT_TEXT_PATH
addressed in https://github.com/bitrise-steplib/steps-xcode-build-for-test/pull/17/commits/836b4611a0205418ea891e17819a6cde479ffa77